build/dockerfiles/Dockerfile

```diff
@@ -32,6 +32,7 @@ RUN apk add --no-cache bash && \
     apk add --no-cache coreutils
 COPY .htaccess README.md /usr/local/apache2/htdocs/
 COPY --from=builder /build/devfiles /usr/local/apache2/htdocs/devfiles
+COPY --from=builder /build/resources /usr/local/apache2/htdocs/resources
```

Comment:
probably we have to add permissions for `/build/resources` folder as it was done for `/build/devfiles` https://github.com/eclipse-che/che-devfile-registry/blob/dab1122603418eecb495fa5744373290e52d9f6a/build/dockerfiles/Dockerfile#L21
To avoid problems on OpenShift like https://github.com/eclipse/che/issues/20880

Reply:
fixed